htdocs/core/lib/functions.lib.php

```diff
@@ -1075,8 +1075,14 @@ function GETPOST($paramname, $check = 'alphanohtml', $method = 0, $filter = null
 				$tmpcheck = $tmparray[1];
 			}
 		}
-		foreach ($out as $outkey => $outval) {
-			$out[$outkey] = sanitizeVal($outval, $tmpcheck, $filter, $options);
+		// avoid warning when the return is array(0 => '')
```

Comment:
I can't see how you can get array(0 => '') here ?
Which value of $out in input do you have for this ?